testing/architecture-test/src/changes/accepted-changes/accepted-public-api-changes.json

```diff
@@ -2481,14 +2505,6 @@
                 "Method has been removed"
             ]
         },
-        {
-            "type": "org.gradle.plugins.ear.EarPluginConvention",
```

Comment:
How comes this was removed?

Reply:
I wish I knew.  I've gotten a few of these strange deletions this cycle.  I'll restore it.